src/main/java/org/generationcp/middleware/dao/RoleDAO.java

```diff
@@ -32,6 +33,12 @@
 					criteria.createAlias("roleType", "roleType");
 					criteria.add(Restrictions.eq("roleType.id", roleSearchDto.getRoleTypeId()));
 				}
+				if(roleSearchDto.getName() != null && StringUtils.isNotBlank(roleSearchDto.getName())) {
```

Comment:
@vmaletta It seems to be ok now but I dont know if this will mutate to be like instead of equals for name. I think that description will not be needed once the validation is fixed.

Reply:
`isNotBlank` already checks for `null` too